HELP.md

```diff
@@ -0,0 +1,19 @@
+# Getting Started
```

Comment:
I think we should delete this in favor of the README.md.

Reply:
Agreed, this was generated.